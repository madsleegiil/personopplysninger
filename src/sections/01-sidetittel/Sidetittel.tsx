import React from "react";
import { Sidetittel } from "nav-frontend-typografi";
import { FormattedMessage } from "react-intl";

const Tittel = () => (
  <div className="sidetittel__container">
    <Sidetittel>
      <FormattedMessage id="sidetittel" />
    </Sidetittel>
  </div>
);
export default Tittel;
