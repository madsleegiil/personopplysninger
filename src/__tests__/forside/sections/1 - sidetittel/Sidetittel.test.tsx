import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";
import nbMessages from "text/nb";
import Sidetittel from "../../../../pages/forside/sections/1-sidetittel/Sidetittel";

jest.mock("react-modal");
jest.mock("utils/validators");

describe("Sidetittel", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <IntlProvider locale={"nb"} messages={nbMessages}>
          <Sidetittel />
        </IntlProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
