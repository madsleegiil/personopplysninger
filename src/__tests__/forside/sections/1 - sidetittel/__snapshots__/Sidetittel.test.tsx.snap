// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidetittel renders correctly 1`] = `
<div
  className="sidetittel__container"
>
  <h1
    className="typo-sidetittel"
  >
    Personopplysninger
  </h1>
</div>
`;
